Java2020/src/lesson201221/homework/RunHomework.java

```diff
@@ -0,0 +1,29 @@
+package lesson201221.homework;
+
+import java.io.PrintStream;
+import java.time.LocalDateTime;
+import java.util.Random;
+import java.util.Set;
+import java.util.TreeSet;
+
+public class RunHomework {
+
+    public static Random random = new Random();
+    public static final int oneMillion = 1000000;
+    public static final int high = oneMillion;
+    public static final int low = -oneMillion;
+    public static final int nanoTimeToSecondsFactor = 1000000000;
+
+    public static void main(String[] args) {
+
+        Set<Integer> set = new TreeSet<>();
+
+        long start = System.nanoTime();
+
+        for(int i = 0; i < oneMillion; i++) set.add(low + random.nextInt((high - low + 1)));
+
+        long end = System.nanoTime();
+

```

Comment:
Actually, you should make 3 steps:
1. create array & fill it with random numbers
2. put everything from the array to the TreeSet
3. put back everything from the set to the array
Then everything in the array will be sorted.

Reply:
Noted!

will fix it